TR123/GameData.cs

```diff
@@ -0,0 +1,464 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Collections.Immutable;
+using System.Diagnostics;
+using System.Linq;
+using LiveSplit.ComponentUtil;
+
+namespace TR123;
+
+public class GameData
+{
+    /// <summary>Used to calculate <see cref="TimeSpan" />s from IGT ticks.</summary>
+    public const int IGTTicksPerSecond = 30;
```

Comment:
This is a holdover from OG splitters, where uppercase acronyms were used and even added to ReSharper dictionary to prevent highlights. That previous behavior is incorrect and it should be fixed for other splitters at some point.